pt-BR/mail/chrome/messenger/preferences/receipts.dtd

```diff
@@ -1,5 +1,5 @@
 <!ENTITY dialog.title                  "Confirmações de leitura">
-<!ENTITY receiptSettings.label         "Return Receipts">
+<!ENTITY receiptSettings.label         "Confirmações de leitura">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY receiptSettings.label         "Confirmações de entrega">